### DISCUSSION TITLE
本地启动Uncaught ReferenceError: antd is not defined 和Cannot read properties of undefined (reading 'extend')

### DISCUSSION THREAD

**Discussion by tangjuncheng1986** [alibaba/designable#377] [Q&A]
<img width="1685" alt="image" src="https://github.com/alibaba/designable/assets/19219876/6b3c70cc-c6e2-45e6-a8a2-7ca78f239d75">

**Comment by tangjuncheng1986** [comment]
最新的master分支，出现这个问题

**Comment by tangjuncheng1986** [comment]
需要把template.ejs中做如下修改：

<script src="https://unpkg.com/antd@4.15.2/dist/antd.min.js"></script>

**Reply by lyce1874** [reply]
4.15.2会导致级联选择器样式出现错乱，引用为4.17,4暂未出现问题

**Comment by jinxueTiT** [comment]
为什么我调整了还是再报那个错呢

**Comment by kikcmime** [comment]
请问这个有解了吗？node版本是多少，呀

**Reply by kikcmime** [reply]
![image](https://github.com/user-attachments/assets/95e81c0d-6a72-40cd-a325-25480b756cbe)
